guides/main_concepts/README.md

```diff
@@ -1,14 +1,14 @@
 # Main Concepts
 
-In this section, we go through the main concepts of MeiliSearch.
+This section defines the main concepts of MeiliSearch.
 
 ::: tip Important
-MeiliSearch is an **asynchronous** API. It means that when it does not behave like a synchronous API when you are dealing with the request's responses. [Read more in the advanced guide](/guides/advanced_guides/asynchronous_updates.md).
+MeiliSearch is an **asynchronous** API. This means that, unlike a synchronous API, it doesn't wait for a return as calls are placed in a queue for processing. For further information please read [the advanced guide](/guides/advanced_guides/asynchronous_updates.md).
```

Comment:
I'm not sure about my suggestion but I'm not convinced by the meaning of yours.
```suggestion
MeiliSearch is an **asynchronous** API. This means that, unlike a synchronous API, it doesn't wait for the call to be done as calls are placed in a queue for processing. For further information please read [the advanced guide](/guides/advanced_guides/asynchronous_updates.md).
```

Reply:
Yes I get it! I commit the change!